internal/artifacts/module_arch_test.go

```diff
@@ -404,6 +404,37 @@ builders:
 				validateArchiveContentsExcludes([]string{"ignore"}, getFullPathInTmpFolder("mta", "htmlapp2", "data.zip"))
 			})
 
+			It("zip module referencing the whole project (path='.')", func() {
```

Comment:
Add tests for the cases of module build without sending target and module build when sending target. These cases use `ModuleLoc` instead of `Loc`.